### ISSUE TITLE
Плагин Kandinsky 2.0 для AUTOMATIC1111

### ISSUE THREAD

**Issue by 0-NiK-0** [ai-forever/Kandinsky-2.0#13]
Здравствуйте! Очень хочется увидеть Kandinsky 2.0 в формате плагина для AUTOMATIC1111 и других популярных интерфейсов Stable Diffusion (ComfyUI, InvokeAI, Easy Diffusion . . .).
Буду очень признателен, если вы это реализуете 😊

**Comment by FurkanGozukara** [comment]
yes automatic1111 web ui extension and safe tensors would be great

so far I made a tutorial for jupyter notebook

27.) Python Script - Jupyter Based - PC - Free
[**Midjourney Level NEW Open Source Kandinsky 2.1 Beats Stable Diffusion - Installation And Usage Guide**](https://youtu.be/dYt9xJ7dnpU)
[![image](https://user-images.githubusercontent.com/19240467/230183162-8a6f7e84-dcd9-45b5-a94c-b93a10778f42.png)
](https://youtu.be/dYt9xJ7dnpU)
